GoogleForCreators/web-stories-wp
# Fix incorrect publish time handling

## Summary
- Ensures that the date is kept in the same timezone when loading and saving the time.
- Displays timezone abbreviation in the calendar, if available.


## Relevant Technical Choices



## To-do



## User-facing changes
Timezone in the calendar. The tooltip doesn't have the best visibility, perhaps we should add some shadow for it in the design system.
![Screenshot 2021-06-02 at 12 50 06](https://user-images.githubusercontent.com/3294597/120459749-c4416f00-c398-11eb-92c8-97d448c473cb.png)



## Testing Instructions
1. Set your timezone in the General Settings of WordPress to something else than UTC 0
2. Go to the editor and open the Document panel publish time calendar.
3. Verify that in the right top corner, the timezone is displayed, either UTC or your timezone.
4. Set the Publish time to something else than it was before.
5. Save and refresh the page
6. Verify the time displayed is the same as you saved previously.


### QA






This PR can be tested by following these steps:

1. See the instructions above.

### UAT







This PR can be tested by following these steps:

1.

## Checklist




---



Fixes #7780
